### PR TITLE
clusterlogging pkg: IsReady method added to the lokistack handler

### PR DESCRIPTION
**Pull request by elenagerman**

### CODE REVIEW

**Comment by elenagerman** [comment]
@trewest @kononovn @achuzhoy @cdvultur PTAL

File: pkg/clusterlogging/lokistack.go [hunk 1]
```diff
@@ -361,6 +364,32 @@ func (builder *LokiStackBuilder) WithTemplate(
 	return builder
 }
 
+// IsReady check if the lokiStack state is Ready.
```
Comment (achuzhoy) [inline pkg/clusterlogging/lokistack.go:367]:
```suggestion
// IsReady checks for the duration of timeout if the lokiStack state is Ready.
```

**Comment by achuzhoy** [comment]
We'll need a test for it too.

**Review by achuzhoy — DISMISSED** [review]

**Review by kononovn — CHANGES_REQUESTED** [review]
looks good. Please add test case for this function

**Review by kononovn — APPROVED** [review]
/lgtm

**Pull request merged**